code/__DEFINES/dcs/signals/signals_client.dm

```diff
@@ -0,0 +1,4 @@
+/// from base of [/client/proc/handle_spam_prevention] (message, mute_type)
+#define COMSIG_CLIENT_AUTOMUTE_CHECK "client_automute_check"
+/// Prevents the automute system checking this client for repeated messages.
+#define SIGVAL_WAIVE_AUTOMUTE_CHECK (1<<0)
```

Comment:
1. Indent this up one so it's obvious it's part of `COMSIG_CLIENT_AUTOMUTE_CHECK`.
2. Typo, "SIGVAL"

Reply:
2 is intentional. We have COMSIG -> COMponent SIGnal. I am adding SIGVAL -> SIGnal VALue.

Reply:
COMSIG needs to be killed into SIGNAL anyway, that's from a time when signals could only be registered from components.

Can you just remove `SIGVAL_` entirely and just have `WAIVE_AUTOMUTE_CHECK`?

Reply:
Sure.